terraform/modules/rds_migration/ecs.tf

```diff
@@ -26,6 +26,8 @@ resource "aws_ecs_task_definition" "db_migration" {
           awslogs-group         = aws_cloudwatch_log_group.this.id
           awslogs-region        = "eu-west-2"
           awslogs-stream-prefix = var.prefix
+          mode                  = "non-blocking"

```

Comment:
I felt that we would want the task to keep running in the event of a log service failure, so I introduced non-blocking mode here (just like in the application module).